spec/controllers/articles_controller_spec.rb

```diff
@@ -10,19 +10,19 @@
       after { PaperTrail.enabled = false }
 
       it "returns true" do
-        expect(PaperTrail.enabled?).to eq(true)
+        expect(PaperTrail.enabled?).to be(true)
```

Comment:
I would like the cop for these confusing `eq` -> `be` to be disabled. I doubt there is any useful difference between them or that people remember which one to use.